# ChangeString.srv
string new_string
---
bool success